code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -70,6 +70,43 @@
 				/obj/item/grenade/clusterbuster,
 				/obj/item/storage/box/syndie_kit/throwing_weapons = 3)
 
+/obj/effect/spawner/lootdrop/prison_contraband
+	name = "prison contraband loot spawner"
+	loot = list(/obj/item/clothing/mask/cigarette/space_cigarette = 4,
+				/obj/item/clothing/mask/cigarette/robust = 2,
+				/obj/item/clothing/mask/cigarette/carp = 3,
+				/obj/item/clothing/mask/cigarette/uplift = 2,
+				/obj/item/clothing/mask/cigarette/dromedary = 3,
+				/obj/item/clothing/mask/cigarette/robustgold = 1,
+				/obj/item/storage/fancy/cigarettes/cigpack_uplift = 3,
+				/obj/item/storage/fancy/cigarettes = 3,
+				/obj/item/clothing/mask/cigarette/rollie/cannabis = 4,
+				/obj/item/toy/crayon/spraycan = 2,
+				/obj/item/crowbar = 1,

```

Comment:
Maybe we could add screwdrivers to this list too? I like the idea of prisoners being able to make grenades and reconstruct destroyed machines.

Glowsticks might also be a nice thing to add to the table.

If you wanted to, you could add wirecutters to this pool as well, making the screwdriver+crowbar+wirecutters prisoner dream theoretically possible, if incredibly unlikely.

Reply:
also, it looks like you've weighted cigarettes a bit too heavily, considering that (unless you have the smoker quirk) a lighter or match for them might not even spawn in the permabrig

Reply:
I'll consider the other tools, but I am kinda fine with just crowbars being a possibility for now. Also, the logic behind the cigarettes was that you really only needed one lighter to light them, and there's a good few chances for one to spawn. I'll look at it again.